### ISSUE TITLE
Login Insecure

### ISSUE THREAD

**Issue by il8677** [ISD-Sound-and-Lights/inventor2#1]
The login is insecure since the passwords are stored inside the open sourced repository avalible to the public, it means people could modify this repositories code to find the correct password. Make it so the hashes and the salts are stored in text files.

**Comment by ghost** [comment]
The only things visible to the public are the salted hashes and the salts. The salted hashes are useless to an attacker. It's not ideal, but it works and does not pose a security risk until a password change utility is added.

**Comment by il8677** [comment]
I will prove, I write program that finds password

**Comment by ghost** [comment]
The only way to find a password from a hash is to use a rainbow table. A salt negates a rainbow table.

**Comment by il8677** [comment]
On the contrary, if you know the password hashes and even have he authentication function code, you can run a brute force program on a local machine. If the password hashes are not visible to everyone, the only way to find the password is to brute force the server, which is detectable and stoppable, unlike running it on a local machine, which is siginficantly faster and completely undetectable

**Comment by ghost** [comment]
You're right. However, as we cannot use the passwords hashed here any longer as anyone with malicious intent could go through the commit history, there is no point changing it for the time being until the configuration files work.

**Comment by il8677** [comment]
Change the passwords

**Comment by il8677** [comment]
Make the passwords changeable and hashed in a secure file

**Comment by ghost** [comment]
As this is not a production system, it is not urgent. The hashes for these passwords are already exposed, why expose another set?

**Comment by ghost** [comment]
I will make the passwords changeable next time I am working on it, or you can if you like (make a new branch first please!)

**Comment by il8677** [comment]
Ok
